### PR TITLE
move common related to validate

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
Ready

## Description
As requested I split the PR
Move validate related modules from common to validate directly.

### CODE REVIEW

File: demisto_sdk/commands/common/git_tools.py [hunk 1]
```diff
@@ -19,7 +19,8 @@ def get_current_working_branch() -> str:
 
 
 def get_changed_files(from_branch: str = 'master', filter_results: Callable = None):
-    temp_files = run_command(f'git diff --name-status {from_branch}').split('\n')
+    temp_files = run_command(
```
Comment (yaakovi) [inline demisto_sdk/commands/common/git_tools.py]: why did you change all those lines?

Comment (GalRabin) [reply demisto_sdk/commands/common/git_tools.py]: It’s the auto pep8 precondition configuration, it spit automatically

**Comment by yaakovi** [comment]
@GalRabin you haven't answered all my questions

**Comment by yaakovi** [comment]
closing this PR for now.
let's redo it again with `git mv` so the files will be marked as R100 (rename only)
